### ISSUE TITLE
Alignment in a resource with a single parameter

### ISSUE THREAD

**Issue by gerases** [puppetlabs/puppet-lint#30]
Should this trigger a linter error:

```
class { 'some_class':
  some_param                             => 'some_value',
}
```

Note the extra spaces.

**Comment by LukasAud** [comment]
Hi @gerases, sorry for the long delay in follow-up.

I have replicated the issue and can confirm this is still active. Given the behaviour of our arrow_alignment plugin, I would say this is probably a bug as adding any extra parameters to the class triggers the correct response (aligning '=>' symbols one space after the longest parameter name).

To answer the question: Yes, it should be throwing a lint warning. We have ticketed this and will be looking to fix it in the near future.
